mods/_testmod/scripts/data/recruits/starwalker.lua

```diff
@@ -6,9 +7,14 @@ function Starwalker:init()
     -- Display Name
     self.name = "Starwalker"
     
-    -- How many times an enemy needs to be spared to be recruited.
+    -- self.name = "Werewerewire"
```

Comment:
Why's this here?

Reply:
mostly for testing stuff related to long names, to be accurate with Deltaune. It's commented.
The virovirokun should be used as a base.